VideoLocker/res/values/strings.xml

```diff
@@ -573,7 +571,7 @@
     <!-- Message on course page for course that hasn't started -->
     <string name="course_content_available_text">This course hasn\'t started yet. Come back on {start_time} to see your videos.</string>
     <!-- Button text, allowing user to open the edx website in a web browser -->
-    <string name="open_edx_btn_text">Open edx.org in my browser</string>
+    <string name="open_destination_btn_text">Open {platform_destination} in my browser</string>
```

Comment:
"open_destination" sounds quite weird… maybe change to "website" or some other easily understandable term? Note that the "edx" wording is present in the description comment as well.